machine/machine_core/ssh_connection.py

```diff
@@ -32,7 +32,7 @@
 # HACK: some projects directly import ssh_connection before adjusting sys.path; add bots root dir
 sys.path.insert(1, os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))))
 
-from lib.constants import TEST_DIR  # noqa: E402
+from lib.constants import TEST_DIR
```

Comment:
According to my understanding, this noqa should still beq required.  Or did ruff (very) recently get smarter about this?